docs/jwst/outlier_detection/outlier_detection_tso.rst

```diff
@@ -0,0 +1,41 @@
+.. _outlier-detection-tso:
+
+Outlier Detection for TSO Data
+==============================
+
+Time-series observations (TSO) result in input data stored as a 3D CubeModel
+where each plane in the cube represents a separate integration without changing the
+pointing.  Normal imaging data benefit from combining all integrations into a
+single image. TSO data's value, however, comes from looking for variations from one
+integration to the next.  The outlier detection algorithm, therefore, gets run with 
+a few variations to accomodate the nature of these 3D data.
+
+#. Input data is converted into a CubeModel (3D data array) if a ModelContainer
+   of 2D data arrays is provided.
+
+#. The median image is created without resampling the input data
+
+   * The median image is created by combining all planes in the 
+     ModelContainer pixel-by-pixel using a rolling-median algorithm, in order
+     to flag outliers frame-by-frame but preserve real time variability.
+   * The ``n_ints`` parameter specifies the number of integrations over
+     which to compute the median. The default is 25. If the number of integrations
+     is less than ``n_ints``, a simple median is used instead.
+   * The ``nlow`` and ``nhigh`` parameters specify how many low and high values
+     to ignore when computing the median for any given pixel.
+   * The ``maskpt`` parameter sets the percentage of the weight image values to
+     use, and any pixel with a weight below this value gets flagged as "bad" and
+     ignored when resampled.
```

Comment:
We aren't doing any resampling for TSO data, so is this bullet relevant?

Reply:
I really meant "when the median is taken" and have changed the text accordingly